SpatialGDK/Documentation/content/handover-between-server-workers.md

```diff
@@ -19,16 +19,22 @@ Note that server-worker authority over properties is different to server-worker
 ## How to facilitate Actor handover
 
 To facilitate an Actor’s property handover between server-workers, follow the instructions below:
-1.  In the Actor’s class, mark the property field with a UPROPERTY `Handover` tag, as shown in the example below.
+1.  In the Actor’s class, mark the property field with a `Handover` tag in the `UPROPERTY` macro, as shown in the example below.
 
 ```
 UPROPERTY(Handover)
 float MyServerSideVariable;
 ```
 
-2. Locate the `DefaultEditorSpatialGDK.ini` config file in the `<ProjectRoot>\<GameRoot>\Config\` directory and add the Actor’s class to its list, as shown in the [example]({{urlRoot}}/content/interop#example) in the Interop Code Generator documentation.
+2. In the Actor's `UCLASS` macro, mark the class as a `SpatialType`, as show in the example below.
 
-1. Run the [Interop Code Generator]({{urlRoot}}/content/interop). This generates the [type bindings]({{urlRoot}}/content/glossary#type-bindings) for your Actor’s class, including the `Handover` bindings.
+```
+UCLASS(SpatialType)
+class AMyActor
+{
```

Comment:
Why this {}

Reply:
To indicate it's not a forward declaration.